pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <groupId>io.curity.identityserver.plugin</groupId>
     <artifactId>identityserver.plugins.authenticationactions.debug</artifactId>
-    <version>1.2.0</version>
+    <version>1.2.1</version>
```

Comment:
is this a revision or a minor?

Reply:
perhaps a minor. but it's just a template change

Reply:
changed it to minor